recipes/imapclient/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{%set name = "IMAPClient" %}
+{%set version = "1.0.2" %}
+{%set compress_type = "tar.gz" %}
+{%set hash_type = "sha256" %}
+{%set hash_val = "3b6e9a5f0484be0151f4106c5aba546eb3a382905ba69e86fa52ce5de95fed3f" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.{{ compress_type }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ compress_type }}
+  {{ hash_type }}: {{ hash_val }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - backports.ssl >=0.0.9
+    - mock ==1.3.0
+    - pyopenssl >=0.15.1
+    - six
```

Comment:
Looks like this needs [sphinx](https://github.com/mjs/imapclient/blob/master/setup.py#L77)